src/routes/github/github-router.ts

```diff
@@ -15,19 +15,19 @@ export const GithubRouter = Router();
 const subRouter = Router({ mergeParams: true });
 GithubRouter.use(`/:uuid(${UUID_REGEX})?`, subRouter);
 
+// Webhook Route
```

Comment:
Just needed to move this bad boy up so webhook events don't hit the ghes app middleware.